Lombiq.HelpfulLibraries/Libraries/ResourceManagement/ResourceFilterBuilder.cs

```diff
@@ -22,12 +22,12 @@ public ResourceFilter When(Func<HttpContext, bool> filter)
         }
 
         public ResourceFilter WhenPath(string path) =>
-            When(context => context.Request.Path.Value.ToUpperInvariant() == path.ToUpperInvariant());
+            When(context => context.Request.Path.Value.EqualsOrdinalIgnoreCase(path));

```

Comment:
Note that here and below the culture also changed form invariant to ordinal. However, since we're talking about a file path, I think it should be ordinal.

Reply:
I think if the file path is in a case-insensitive file system (or if it's a [custom route literal](https://docs.microsoft.com/en-us/aspnet/core/fundamentals/routing?view=aspnetcore-3.1#route-template-reference)) then this could cause problems.

Reply:
What kind of problems do you mean? Note that I didn't change the case insensitive behavior, that was how it worked before too.

Thinking about it, we should aim for cross-platform, thus file paths rather need to be case-sensitive. Talking about URLs in ASP.NET Core this is the case by default as well, even if it's an URL for a static file, if hosted under IIS (not sure what it does behind the scenes but this can cause amusing issues when a path with incorrect casing will work for a file locally on Windows when run via IIS Express but won't work when deployed to an Azure App Service, even if hosted on Windows). But this is not a route but a static file path anyway.

Reply:
Apologies, for some reason I misread it and thought that you've turned it into just ordinal.